README.md

```diff
@@ -41,10 +41,14 @@ parent:
 </div>
 
 
+Cosmos-SDK is a framework for building blockchain applications in Golang.
 
+Cosmos-SDK has been used to build a number of high value public blockchains, including:
 
-The Cosmos-SDK is a framework for building blockchain applications in Golang.
-It is being used to build [`Gaia`](https://github.com/cosmos/gaia), the first implementation of the Cosmos Hub.
+* [`Gaia`](https://github.com/cosmos/gaia), the original Cosmos Hub.

```

Comment:
I don't think we should add any chains. We should just direct people to the awesome repo. Thoughts?

Reply:
I think that the awesome repo is a great place to point people to!